pkgs/top-level/all-packages.nix

```diff
@@ -8432,6 +8432,8 @@ with pkgs;
   };
   libkrb5 = krb5Full.override { type = "lib"; };
 
+  languageMachines = import ../development/libraries/languagemachines/packages.nix { inherit callPackage; };
```

Comment:
As it is not part of the main namespace, the packages will be not build by hydra. Are you ok with this?